### PR TITLE
chore/codeowners:added codeowners file

### PR DESCRIPTION
**Pull request by calumcraig**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @dirvine

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: codeowners [hunk 1]
```diff
@@ -0,0 +1 @@
+* ustulation
```
Comment (S-Coyle) [inline codeowners]: @calumcraig Should "ustulation" have an @ in front of it?

Comment (calumcraig) [reply codeowners]: Yes, thanks for catching that - copy and paste fatigue setting in.

**Review by S-Coyle — APPROVED** [review]
👍

**Pull request merged**